tests/support.py

```diff
@@ -290,6 +292,56 @@ def cleanpath(relative_path, test_case, ensure_dir=False):
     return tmp_path
 
 
+def fixturefile(relative_path, fixture_format=None):
+    """Support function for loading fixtures from their serialised format.
+
+    Doing so is a little more involved than it may seem because serialisation
+    formats may not capture various nuances of the python data they represent.
+    For this reason each supported format defers to a format specific function
+    which can then, for example, load hints about
```

Comment:
This docstring sentence stops rather abruptly. I'll see if I can guess an ending, but feel free to follow-up.

Reply:
It's honestly been so long that I can't remember what this was meant to have been and certainly not why it abruptly stopped unfortunately - at this point whatever you dreamt up is probably as good as anything I could so will follow-up if anything bubbles back to mind.